src/Tribe/Views/V2/Views/Widgets/Widget_View.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * The Base Widget View.
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Views\Widgets;
+
+use Tribe\Events\Views\V2\View;
+
+/**
+ * Class Widget_View
+ *
+ * @since   5.2.1
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ */
+class Widget_View extends View {
+
+	/**
+	 * Overrides the base View method.
+	 *
+	 * @since TBD
+	 *
+	 * @return array<string,mixed> The Widget View template vars, modified if required.
+	 */
+	protected function setup_template_vars() {
+		$template_vars = parent::setup_template_vars();
+
+		$template_vars['container_classes'] = $this->get_html_classes();
+
+		return $template_vars;
+	}
+
+	/**
+	 * {@inheritDoc}
```

Comment:
:no_entry_sign: Doc comment for parameter "$classes" missing

<sub>hash: e9da3b783c3bf4646ba05e0f21b15100</sub>